fe/src/project/Account/areas/Accounts/components/SignIn/SignInForm.js

```diff
@@ -60,14 +60,7 @@ function SignInForm(props) {
                             value={password} onChange={handleOnChange} helperText={dirty.password ? errors.password : ""} error={(errors.password.length > 0) && dirty.password} />
                     </div>
                     <div className="is-flex is-justify-content-end">
-                        <button 
-                            className="button is-text"
-                            onClick={(e) => {
-                                e.preventDefault();
-                                NavigationHelper.redirect(props.resetPasswordUrl)
-                            }}>
-                                {props.forgotPasswordLabel}
-                        </button>
+                        <a className="button is-text is-size-7" href={props.resetPasswordUrl}>{props.forgotPasswordLabel}</a>
```

Comment:
Hi @SebaKas21 , according to the W3C it makes no sense to use a button in HTML syntax when it's actually a link.